src/subcommands/subcommand.go

```diff
@@ -84,16 +84,12 @@ func baseHelp() {
 	fmt.Println("\nUse 'semantifly <subcommand> --help' for more information about a specific subcommand.")
 }
 
-func setupSemantifly() {
-	semantifly_dir := flag.String("semantifly_dir", os.ExpandEnv("$HOME/.semantifly"), "Where to read existing semantifly data, and write new semantifly data.")
-	semantifly_index := flag.String("semantifly_index", "default", "By default, semantifly writes data to the 'default' subdir of the configured semantifly_dir. Setting this to a value other than 'default' allows for multiple indices on the same local machine.")
```

Comment:
Why are these not configurable any more? Don't we just need to set these to better defaults and remove the index_path subcommand args (eg from executeAdd)?

Reply:
never mind, per discussion, it's ok to set the index path in the subcommand args instead